app/assets/stylesheets/variables/_agaricus.scss

```diff
@@ -9,59 +26,78 @@ $xanthodermus_stain:   #D4A833;
 $polyozellus_light:    #a5bffa;
 $polyozellus_dark:     #5167b2;
 
-$BUTTON_FG_COLOR:             $cupreobrunneus_gills;
-
-$BODY_FG_COLOR:               $cupreobrunneus_gills;
-$BODY_BG_COLOR:               $campestris_cap;
-
-$LINK_FG_COLOR:               $brasiliensis_gills_1;
-$LINK_VISITED_FG_COLOR:       $cupreobrunneus_gills;
-$LINK_HOVER_FG_COLOR:         $brasiliensis_gills_2;
-$LINK_HOVER_BG_COLOR:         $campestris_cap;
-
-$LOGO_BORDER_COLOR:           $cupreobrunneus_gills;
-$LOGO_FG_COLOR:               $cupreobrunneus_gills;
-$LOGO_BG_COLOR:               $semotus_cap;
-$LOGO_HOVER_FG_COLOR:         $brasiliensis_gills_2;
-$LOGO_HOVER_BG_COLOR:         $campestris_cap;
-
-$LEFT_BAR_BORDER_COLOR:       $augustus_cap;
-$LEFT_BAR_HEADER_FG_COLOR:    $cupreobrunneus_gills;
-$LEFT_BAR_HEADER_BG_COLOR:    $augustus_cap;
-$LEFT_BAR_FG_COLOR:           $brasiliensis_gills_1;
-$LEFT_BAR_BG_COLOR:           $augustus_cap;
-$LEFT_BAR_HOVER_FG_COLOR:     $brasiliensis_gills_2;
-$LEFT_BAR_HOVER_BG_COLOR:     $campestris_cap;
-
-$TOP_BAR_FG_COLOR:            $cupreobrunneus_gills;
-$TOP_BAR_BG_COLOR:            $campestris_cap;
-$TOP_BAR_LINK_FG_COLOR:       $brasiliensis_gills_1;
-$TOP_BAR_LINK_HOVER_FG_COLOR: $brasiliensis_gills_2;
-$TOP_BAR_LINK_HOVER_BG_COLOR: $campestris_cap;
-
-$LIST_BORDER_COLOR:           transparent;
-//$LIST_BORDER_COLOR:           #DDDDDD;
-$LIST_HEADER_FG_COLOR:        #777777;
-$LIST_HEADER_BG_COLOR:        #EEEEEE;
-$LIST_FG_COLOR:               $cupreobrunneus_gills;
-$LIST_BG_COLOR:               mix($campestris_cap, white, 50%);
-
-$TOOLTIP_FG_COLOR:            $campestris_cap;
-$TOOLTIP_BG_COLOR:            $cupreobrunneus_gills;
-
-$PAGER_FG_COLOR:              $cupreobrunneus_gills;
-$PAGER_HOVER_FG_COLOR:        $brasiliensis_gills_2;
-$PAGER_HOVER_BG_COLOR:        $campestris_cap;
-$PAGER_ACTIVE_FG_COLOR:       $campestris_cap;
-$PAGER_ACTIVE_BG_COLOR:       $brasiliensis_gills_2;
-
-$MENU_BORDER_COLOR:           $brasiliensis_gills_2;
-$MENU_FG_COLOR:               black;
-$MENU_BG_COLOR:               white;
-$MENU_HOT_FG_COLOR:           black;
-$MENU_HOT_BG_COLOR:           $xanthodermus_stain;
-$MENU_WARM_FG_COLOR:          black;
-$MENU_WARM_BG_COLOR:          $lilaceps_stain;
+// ==================================================
+// SECTION 2: SEMANTIC COLOR ASSIGNMENTS
+// ==================================================
+
+$PRIMARY_COLOR:                     $brasiliensis_gills_1;
+$SECONDARY_COLOR:                   $augustus_cap;
+$SUCCESS_COLOR:                     #70C070;
```

Comment:
This should be defined higher up as something like `$agaricus_green` or `$default_green`.

Reply:
It looks like this came in from _defaults.  Having an easy way to use the default colors makes sense to me.  In defaults this is defined as:

`$palette_success_green:             #70C070;`

I don't know if it would work to do:

`$SUCCESS_COLOR: $palette_success_green`

Or if we should have something like `$default_success_green`.  The problem is that this can lead us back to where we started where we simple don't define `SUCCESS_COLOR` color in this theme, but instead rely on the defaults to fill in the values.  It's a classic problem of different critieria for what makes something "simple".